keymapviz/keyboards/planck/__init__.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+
+keymap_keyword = '(?:LAYOUT_planck_grid|LAYOUT_planck_mit|LAYOUT_ortho_4x12|LAYOUT_ortho_1x2u)'
```

Comment:
```suggestion
keymap_keyword = '(?:LAYOUT_planck_grid|LAYOUT_planck_mit|LAYOUT_ortho_4x12|LAYOUT_planck_1x2uC)'
```

I couldn't find `LAYOUT_ortho_1x2u` , so I thought `LAYOUT_planck_1x2uC` would be correct.